### ISSUE TITLE
Support date in feeds storage uri params

### ISSUE THREAD

**Issue by honzajavorek** [scrapy/scrapy#5387]
## Summary

I'd like to store feeds per day. There's only `time`, but that's not suitable for my use case. If the storage URI parameters supported formatting by `.format()`, perhaps I could use something along the lines of `{time!%Y}`, but that's not supported either. The only way to put day into the URI seems to be to hardcode it like this:

```python
from datetime import date

today = date.today()
FEEDS = {
    f'{FEEDS_DIR}/{today:%Y}/{today:%m}/{today:%d}-%(name)s.jsonl': {
        'format': 'jsonlines',
        'indent': 2,
        'overwrite': True,
        'item_export_kwargs': {
           'ensure_ascii': False,
           'sort_keys': True,
        },
    },
}
```

## Motivation

I run scrapers once per day and want to overwrite files, which are of the same day, then store them indefinitely sorted by days. Putting time to the URI is too fine-grained and wouldn't result in overwriting files made on the same day.

**Comment by honzajavorek** [comment]
I just realized I could have a spider base class with a parameter `today`, which would be populated in something like `from_crawler()`, but that seems to be very convoluted.

**Comment by elacuesta** [comment]
Please see [`FEED_URI_PARAMS`](https://docs.scrapy.org/en/2.5/topics/feed-exports.html#feed-uri-params). Be advised of #4962, which was fixed by #4966 but not yet released.

**Comment by honzajavorek** [comment]
Oh I missed that one. Thank you! ❤️
